code/game/objects/items/weapons/implants/implant_explosive.dm

```diff
@@ -133,6 +133,8 @@
 /obj/item/weapon/implant/dust
 	name = "duster implant"
 	desc = "An alarm which monitors host vital signs, transmitting a radio message and dusting the corpse on death."
+	icon = 'icons/effects/blood.dmi'
+	icon_state = "remains"
 
```

Comment:
@tigercat2000 
Explosive implants already have an explosion icon, for their HUD. So, they probably look like an explosion if extracted via surgery. Yet, this is not considered an issue, because it never happens. Even if it was an issue, given that dust implants are not meant to be re-used, the obvious solution would be to make them qdel themselves on extraction. Unless you have a better suggestion that correctly gives them an icon on the HUD, without changing their in-game icon?